src/io/ImportElement.hh

```diff
@@ -18,21 +18,22 @@ namespace celeritas
 /*!
  * Store element data.
  *
- * Used by ImportMaterial and GdmlGeometryMap.
- *
- * The data is exported via the app/geant-exporter. For further expanding this
- * struct, add the aproppriate variables here and fetch the new values in
- * \c app/geant-exporter.cc : store_geometry(...).
+ * The data is exported via the \e app/geant-exporter. For further expanding
+ * this struct, add the aproppriate variables here and fetch the new values in
+ * \c geant-exporter.cc:store_geometry(...).
  *
  * Units are defined at export time in the aforementioned function.
+ *
+ * \sa ImportData
  */
 struct ImportElement
 {
-    std::string name;
-    int         atomic_number;
-    real_type   atomic_mass;           // [atomic mass unit]
-    real_type   radiation_length_tsai; // [g/cm^2]
-    real_type   coulomb_factor;
+    unsigned int element_id;
```

Comment:
Why is this new field necessary? Isn't it just the index of this element in the vector of `ImportElement`s?